src/crt.c

```diff
@@ -11,31 +11,33 @@
 #include <aws/http/http.h>
 
 struct aws_allocator *aws_crt_allocator(void) {
-  return aws_default_allocator();
+    return aws_default_allocator();
 }
 
 void aws_crt_init(void) {
-  struct aws_allocator *allocator = aws_crt_allocator();
-  aws_common_library_init(allocator);
-  aws_io_library_init(allocator);
-  aws_compression_library_init(allocator);
-  aws_http_library_init(allocator);
-  aws_cal_library_init(allocator);
-  aws_auth_library_init(allocator);
+    struct aws_allocator *allocator = aws_crt_allocator();
+    aws_common_library_init(allocator);
+    aws_io_library_init(allocator);
+    aws_compression_library_init(allocator);
+    aws_http_library_init(allocator);
+    aws_cal_library_init(allocator);
+    aws_auth_library_init(allocator);
```

Comment:
is s3 going here or later?

Reply:
Later, possibly in its own additional module